RFCs/0014-ToJsonString.md

```diff
@@ -0,0 +1,113 @@
+# New Intrinsic Function to Convert Template Block to JSON String
+
+* **Original Author(s):**: @mluk-aws
+* **Tracking Issue**: [Tracking Issue](https://github.com/aws-cloudformation/cfn-language-discussion/issues/14)
+
+# Summary
+
+We will support an intrinsic function called `Fn::ToJsonString` that enables developers to convert a template block into an escaped JSON string, which can be used as input values to string-type properties of CloudFormation resources.
+
+# Examples
+
+Here is an example of a resource containing a string-type property using a JSON formatted string.
+```json
+"MyDashboard": {
+    "Type": "AWS::CloudWatch::Dashboard",
+    "Properties": {
+        "DashboardBody": "{\"start\":\"-PT6H\",\"periodOverride\":\"inherit\",\"widgets\":[{\"type\":\"text\",\"x\":0,\"y\":7,\"width\":3,\"height\":3,\"properties\":{\"markdown\":\"Hello world\"}}]}"
+    }
+}
+```
+
+With the new `Fn::ToJsonString` intrinsic function we can simplify to the following:
+
+## JSON
+```json
+"MyDashboard": {
+    "Type": "AWS::CloudWatch::Dashboard",
+    "Properties": {
+        "DashboardBody": {
+            "Fn::ToJsonString": {
+                "start": "-PT6H",
+                "periodOverride": "inherit",
+                "widgets": [
+                    {  
+                        "type": "text",
+                        "x": 0,
+                        "y": 7,
+                        "width": 3,
+                        "height": 3,
+                        "properties": { "markdown": "Hello world" }
+                    }
+                ]
+            }
+        }
+    }
+}
+```
+
+## YAML
+```yaml
+MyDashboard:
+    Type: AWS::CloudWatch::Dashboard
+    Properties:
+        DashboardBody:
+            Fn::ToJsonString:
+                start: "-PT6H"
+                periodOverride: inherit
+                widgets:
+                    - type: text
+                    x: 0
+                    y: 7
+                    width: 3
+                    height: 3
+                    properties:
+                        markdown: Hello world
+```
+
+### Short Form Syntax
+```yaml
+MyDashboard:
+    Type: AWS::CloudWatch::Dashboard
+    Properties:
+        DashboardBody:
+            !ToJsonString:
+                start: "-PT6H"
+                periodOverride: inherit
+                widgets:
+                    - type: text
+                    x: 0
+                    y: 7
+                    width: 3
+                    height: 3
+                    properties:
+                        markdown: Hello world
+```
+
+# Motivation
```

Comment:
I think we decided to change our template to move the motivation section above examples